content/enable-hibernation.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Enable Hibernation (Suspend to Disk)
+description: >
+    How to partition your drive and set up hibernation in Pop!_OS.
+keywords:
+  - Suspend
+  - Suspend to Disk
+  - Hibernate
+  - Hibernation
+  - Swap
+  - Swap Partition
+  - Encrypted Swap
+  - Encryption
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+## State of Hibernation on Pop!_OS
+
+Hibernation is not currently enabled on Pop!_OS by default. The factors for this choice are:
+
+- Pop!_OS' Default Partition Layout
+- Use of a non-persistent encryption key for SWAP.
+- Additional drive I/O used on Solid State Drives (SSDs)
+- Depending on alloted RAM in a system, and size of NVMe/SSD drive(s), hibernation could add notable delay on resuming a hibernation session and wear out the OS drive prematurely.
```

Comment:
Drop wear comment as it's part of the item above; size of swap space used should be a single item and delay should be it's own item.

Reply:
Wear has been removed, delay retained.